.github/workflows/release.yaml

```diff
@@ -113,9 +113,10 @@ jobs:
           tags: type=ref,event=tag
 
       - name: manifest
-        run: make release-${{ matrix.target }}
-        env:
-          TAG: ${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
+        run: |
+          BOOTSTRAP_IMG_TAG=$RELEASE_TAG
```

Comment:
this RELEASE_TAG envvar comes from github actions?

Reply:
Yeah it's set above but could probably be solved in a better way?